packages/openapi-request-coercer/test/data-driven/coerce-form-data-openapi3.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  args: {
+    parameters: [],
+    requestBody: {
+      required: true,
+      content: {
+        'application/x-www-form-urlencoded': {
+          schema: {
+            type: 'object',
+            properties: {
+              intField: {
+                type: 'integer',
+              },
+              boolField: {
+                type: 'boolean',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+
+  request: {
+    method: 'post',
+    path: '/',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: {
```

Comment:
so you're expecting that express-openapi and koa-openapi will have form data parsing to this is an object not a form urlencoded string?

Reply:
i'm not 100% sure I understand the question, but what I am doing is following the [OpenAPI v3 spec](https://swagger.io/docs/specification/describing-request-body/) where it says:

> In OpenAPI 3.0, form data is modelled using a `type: object` schema where the object properties represent the form fields

Reply:
in my express, I have the form parsing setup this way:

```
const bodyParser = require('body-parser');

...

app.use(bodyParser.urlencoded())
```

Reply:
The new tests you had me add probably answer your question. You send a normal form data request, i.e. 

https://github.com/kogosoftwarellc/open-api/pull/711/files#diff-d3cc19204b178f8d94644ffcb04dcb5f1148d4fee66cdfdb4d7913db449e180f